docs/_documentations/remote-removing.md

```diff
@@ -66,18 +64,19 @@ INFO[0001] Kubernetes namespace: CWCTL will not remove the namespace automatical
 use 'kubectl delete namespace mycodewind-ns' if you would like to remove it
 ```
 
->Since it is possible to install multiple Codewind servers into a Kubernetes namespace the cwctl will not remove the namespace as part of the uninstall. You should only remove the namespace if you no longer need it and are sure there are no other applications installed inside it. For now we will leave the namespace in place.
-
+Since it is possible to install multiple Codewind servers into a Kubernetes namespace, the `cwctl` command does not remove the namespace as part of the uninstall. You should only remove the namespace if you no longer need it and are sure there are no other applications installed inside it. 
 
 ## Removing Keycloak service
 
 Keycloak can be installed alongside a single Codewind deployment or shared by many other Codewind services.
 
 WARNING: Removing Keycloak breaks all remote deployments of Codewind that are using it for authentication. Users will no longer be able to login and will require a full redeploy to recover.
 
-If you are ready to remove Keycloak, use the following `cwctl` command:
+To remove Keycloak, use the following example `cwctl` command:
 
-enter the command and review the results:
+`cwctl remove keycloak --namespace <namespace> --workspace <workspaceID>`
+
+For example:
```

Comment:
As a user, I might be a little confused if I came across two commands both referred to as an "example." What if we removed the example on line 77 and just explained the variables of the command in line 82?

"To remove Keycloak, refer to the following example `cwctl` command, where `mycodewind-ns` is the name of your namespace and `k55333j0` is your workspace ID:"

Reply:
Removed example from line 75.